.slashrc

```diff
@@ -240,8 +240,9 @@ class MediaPlugin(slash.plugins.PluginInterface):
     if slash.config.root.parallel.worker_id is None:
       capture = self.syscapture.checkpoint()
       hangmsgs = [
-        r"\[.*\] i915 0000:00:02.0: Resetting .* after gpu hang",
-        r"\[.*\] i915 0000:00:02.0: Resetting .* for hang on .*",
+        r"\[.*\] i915 0000:00:0[2-5].0: \[drm\] .*GPU HANG",
+        r"\[.*\] i915 0000:00:0[2-5].0: \[drm\] .*GPU hang",
+        r"\[.*\] i915 0000:00:0[2-5].0: \[drm\] Resetting .* time out",
```

Comment:
The `0000:00:0[2-5].0` regular expression (RE) doesn't make sense to me.  Traditionally, the Intel iGPU is always on `0000:00:02.0`.  And AFAICT, the dGPU (e.g. DG1) is on `0000:03:00.0`, which is not captured by this RE (nor the original RE).

Perhaps `0000:00:02.0` in the RE should be replaced with `.*` instead?

Reply:
Also, I think we still want the original RE message, right?  I mean, we want to capture all resets, whether they succeed or not.

Reply:
1. yes, you are right, so we just add new PCI id for dGPU?
2. I forget to check older DUT, which without the kernel upgrade, I will update this PR

Reply:
recover form original and remove PCI id detect